packages/env/lib/env.js

```diff
@@ -477,22 +480,26 @@ async function configureWordPress( environment, config ) {
 	}
 
 	// Activate all plugins.
-	for ( const pluginSource of config.pluginSources ) {
-		await dockerCompose.run(
-			environment === 'development' ? 'cli' : 'tests-cli',
-			`wp plugin activate ${ pluginSource.basename }`,
-			options
-		);
+	if ( Array.isArray( config.pluginSources ) ) {
```

Comment:
Ah, hm. It's unfortunate that plugins and themes aren't automatically activated if you mount the whole directory. Is there a workaround? Maybe we could activate all plugins using e.g. `wp plugin activate --all`? And the first theme using e.g. `wp theme activate $( wp theme list | head -1 )`? Will need to investigate what the implications are of doing this, though.

Reply:
That would probably be a cleaner way to support this. The only other way I can think would be to read all the subdirectory basenames and activate them one by one... I feel like we want to avoid that, but I guess I don't have a strong reason to. 

What implications would there be running those commands compared to just activating them one by one?

Reply:
I have it working essentially how you mentioned. However, I couldn't get `wp theme activate $( wp theme list | head -1 )` working because docker was complaining about too many positional arguments. I did get it working in a less efficient way, but I'll try to put it all in one command :)

Reply:
Yeah, for some reason piping won't work with docker compose run. I'm not sure why.

Try this command: `wp-env run cli "wp theme list --fields=name | head -2"`. It gives the error: `Too many positional arguments: | head -2`. 

However, this does work: `wp-env run cli wp theme list --fields=name | head -2` (I assume because we're piping locally instead of inside the docker container)

Reply:
Ah, right. I think it doesn't work because node runs `docker` without bash (which is what handles things like `$()` and `|`) being involved at all. What you did makes sense 👍